psi4/src/psi4/ccenergy/ccenergy.cc

```diff
@@ -341,6 +341,13 @@ double CCEnergyWavefunction::compute_energy()
         update();
         checkpoint();
     }  // end loop over iterations
+
+    // DGAS Edit
+    Process::environment.globals["CC T1 DIAGNOSTIC"] = moinfo_.t1diag;
+    Process::environment.globals["CC D1 DIAGNOSTIC"] = moinfo_.d1diag;
+    Process::environment.globals["CC NEW D1 DIAGNOSTIC"] = moinfo_.new_d1diag;
+    Process::environment.globals["CC D2 DIAGNOSTIC"] = moinfo_.d2diag;
+
```

Comment:
Id actually push back on this and say lets do it at a large scale when we get to it. Id rather not have a half baked transition and would like to do entire modules at the same time.